otbenchmark/ReliabilityProblem60.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May  5 12:56:49 2020
+
+@author: Jebroun
+
+Class to define the ReliabilityProblem60 benchmark problem.
+"""
+
+from otbenchmark.ReliabilityBenchmarkProblem import ReliabilityBenchmarkProblem
+import openturns as ot
+
+
+class ReliabilityProblem60(ReliabilityBenchmarkProblem):
+    def __init__(
+        self,
+        threshold=0.0,
+        mu1=2200,
+        sigma1=220,
+        mu2=2100,
+        sigma2=210,
+        mu3=2300,
+        sigma3=230,
+        mu4=2000,
+        sigma4=200,
+        mu5=1200,
+        sigma5=480,
+    ):
+        """
+        Creates a reliability problem RP60.
+
+        The event is {g(X) < threshold} with
+
+        you can use this line :
```

Comment:
The description of the problem is unfinished. The "you can use this line" text should be removed.

Reply:
what do you suggest because the formulation of g is complex ?

Reply:
Ok, I get it : you estimated that the formula is too complex so that a better solution is to provide the link to the RPREPO. This makes sense, but has the drawback of not being uniform across the functions.

I suggest to do both for all problems:
* The description of the function is done as a text based on a simplification of the symbolic program. All we have to do is to remove the "var" keywords and this provides a consistent description.
* A link to the RPREPO is inserted in the text.

Please create a separate PR for the second point, so that all the existing problems have a link to the corresponding section of the RPREPO. With this method, the current PR remains as small as possible, focusing on new problems. Otherwise, we may as well never finish this PR!

Reply:
for the first point, that's how I should do it !!!

![capture1](https://user-images.githubusercontent.com/43602056/81938611-7030e480-95f5-11ea-900c-2a819cef8c2a.PNG)

Reply:
![Capture2](https://user-images.githubusercontent.com/43602056/81938840-c7cf5000-95f5-11ea-98c9-9bb25a23b8a4.PNG)

Reply:
Nice.